target/linux/ipq806x/image/Makefile

```diff
@@ -81,6 +81,22 @@ define Device/ZyXELImage
 	IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-to $$$$(BLOCKSIZE) | sysupgrade-tar rootfs=$$$$@ | append-metadata
 endef
 
+define Device/askey_rt4230w-rev6
+	$(call Device/LegacyImage)
+	DEVICE_VENDOR := Askey
+	DEVICE_MODEL := RT4230W
+	DEVICE_VARIANT := REV6
+	SOC := qcom-ipq8065
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	BOARD_NAME := rt4230w-rev6
```

Comment:
If you get a warning during sysupgrade when you remove this line, it might be because your (old) installed system is already using it. Please try forcing sysupgrade once and check out it sill requires BOARD_NAME.

Reply:
Do you mean just remove the BOARD_NAME line entirely?

Reply:
Right, it should not be needed on newly added devices.